index.js

```diff
@@ -72,43 +76,65 @@ const proposerReward = argv.prepaidProposerReward ? argv.prepaidProposerReward :
   );
   const decimals = (await collateral.methods.decimals().call()).toString();
 
+  // Get the final fee for the collateral type to use as default proposer bond.
+  const storeAddress = await getAddress("Store", networkId);
+  const store = new web3.eth.Contract(
+    getAbi("Store"),
+    storeAddress
+  );
+  const finalFee = (await store.methods.computeFinalFee(argv.collateralToken).call()).toString();
+  console.log("final fee:", finalFee);
+  const proposerBond = argv.optimisticOracleProposerBond ? argv.optimisticOracleProposerBond : finalFee;
+
+  // Set FPL.
+  const fpl = await getAddress(argv.fpl + "LongShortPairFinancialProductLibrary", networkId);
```

Comment:
Setting this way will mean that this will error out any time someone tries to pass an address rather than specifying the name - since it will still try to run getAddress with an undefined name.

Reply:
will fix!